azure_arc_servers_jumpstart/docs/policies_mma.md

```diff
@@ -0,0 +1,73 @@
+# Overview
+
+The following README will guide you on how to use Arc for servers to assign Azure Policies to VMs outside of Azure, wether they are on-premise or other clouds. With this feature you can now use Azure Policies to audit settings in the operating system of an Azure Arc connected servers, if a setting is not compliant you can also trigger a remmediation task. 
+
+In this case we will assign a policy to audit if the Azure Arc connected machine has the MMA agent installed, if not, we will use the extensions feature to automatically deploy it to the VM, an enrollment experience that levels to Azure VMs. This approach can be used to make sure all your servers are onboarded to services such as: Azure Monitor, Azure Security Center or Azure Sentinel. 
+
+You can use the Azure Portal, an ARM template, PowerShell script to assign policies to Azure Subscriptions or Resource Groups. In this guide, you will use an ARM template to assign built-in policies. 
+
+**Note: This guide assumes you already deployed VMs or servers that are running on-prem or other clouds and you have connected them to Azure Arc. If you haven't, this repository offers you a way to do so in an automated fashion using either [GCP Ubuntu VM](gcp_terraform_ubuntu.md), [GCP Windows VM](gcp_terraform_windows.md), [AWS Ubuntu VM](aws_terraform_ubuntu.md), [VMware Ubuntu VM](vmware_terraform_ubuntu.md), [VMware Windows Server VM](vmware_terraform_winsrv.md), [Local Ubuntu VM](local_vagrant_ubuntu.md) or [Local Windows VM](local_vagrant_windows.md)**
+
+# Prerequisites
+
+* Clone this repo
+
+* As mentioned, this guide starts at the point where you already deployed and connected VMs or servers to Azure Arc.
+
+    ![](../img/vm_policies/01.png)
+
+    ![](../img/vm_policies/02.png)
```

Comment:
Please align with the rest of the guide naming convention for both Resource Group, i.e "Arc-Servers-Demo" and Arc machines, i.e "Arc-GCP-Demo"